tests/14account/01change-password.pl

```diff
@@ -0,0 +1,220 @@
+my $password = "my secure password";
+
+
+test "After changing password, can't log in with old password",
+   requires => [ local_user_fixture( password => $password ) ],
+
+   check => sub {
+      my ( $user, ) = @_;
+
+      do_request_json_for( $user,
+         method => "POST",
+         uri    => "/r0/account/password",
```

Comment:
This block of functionality seems to be copypasted between most of the tests. Could you pull it out into a `matrix_set_password()` function?